### ISSUE TITLE
`bookmark-set` throws `image-display-size: Invalid image specification: (space :width (1615) :height (2013))`

### ISSUE THREAD

**Issue by NightMachinery** [dalanicolai/image-roll.el#15]
`bookmark-set` throws `image-display-size: Invalid image specification: (space :width (1615) :height (2013))`.

`pdf-view-position-to-register` and `pdf-view-bookmark-make-record` throw the same error. You might need to scroll a bit to get the error.

I am using `better-jumper-mode`, too, and it too cannot jump to positions in PDFs.  (No error messages there, the page just turns gray and on redisplay the position has not changed.)

**Comment by dalanicolai** [comment]
Indeed, I have also not implemented this functionality. However, I found that the functionality for storing the bookmark could easily be implemented/fixed for image-roll and I have pushed a fix (to my pdf-roll branch of pdf-tools). From a quick test here, it looks like it might have fixed the issue. However, it could be that to 'thoroughly' fix it, we might have to update the 'bookmark jump handler' also. Anyway, you can try it out by syncing with the pdf-roll branch of my pdf-tools fork.
